build/buildRunTimeImageBases.sh

```diff
@@ -52,13 +52,6 @@ docker build \
     -t "$RUNTIME_BASE_IMAGE_NAME-stretch" \
     $REPO_DIR
 
-docker build \
-    --pull \
-    --build-arg DEBIAN_FLAVOR=buster \
```

Comment:
I'm curious why we're removing the `docker build` here for the buster runtime base image but keeping the stretch base image above -- are there still stretch images that we're building but none for buster?

Reply:
That's good point, I keep the stretch docker build here since from node 4 to node 12, those images are still stretch based, I'm not sure if we'd like to migrate those versions to bullseye based one? [daniv-msft](https://github.com/daniv-msft) I'm not sure if Antares team is still using these old versions?

Reply:
Currently confirm that node 4 to node 12 are stretch based and node 14, 16 are bullseye based, so we don't have any buster based images for node, and we're removing it from the pipelines as well.